g11n-ws/modules/md-restful-api/src/main/java/com/vmware/vip/api/rest/APIParamValue.java

```diff
@@ -30,6 +30,8 @@
 	public final static String REGION         = "a string which represents region, e.g. US, PT, CN";
 	public final static String COMBINE        = "an integer which represents combine type number 1 or 2";
 	public final static String SCOPE_FILTER   = "a String for filtering the pattern data, separated by comma and underline. e.g. 'dates_eras,dates_dayPeriodsFormat'";
+	public final static String DEFAULT_TERRITORY = "a boolean which represents get the default territory timezone name or not. (default is true)";
```

Comment:
a boolean value to get the default territory timezone name or not, default is true.

Reply:
done